core/src/main/java/io/questdb/std/fastdouble/FastDoubleParser.java

```diff
@@ -0,0 +1,218 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.std.fastdouble;
+
+import io.questdb.std.NumericException;
+
+/**
+ * Provides static method for parsing a {@code double} from a
+ * {@link CharSequence}, {@code char} array or {@code byte} array.
+ */
+public final class FastDoubleParser {
```

Comment:
nit: we should change access modifiers for all other classes in this package to package private to make it clear that `FastDoubleParser` is the entry point. Same consideration applies to `FastFloatParser`

Reply:
done in https://github.com/questdb/questdb/pull/2527/commits/c84fbc9604fa6791384002d3b9a1b38b41759d9d